app/src/main/java/com/example/localieapp/adapter/EarnGridAdapter.kt

```diff
@@ -67,5 +68,15 @@ class EarnGridAdapter(private val context: Context, private val dataset: List<Co
                     .into(holder.imageView)
             }
         }
+        holder.itemView.setOnClickListener(View.OnClickListener() {
+            Log.d("onBindViewHolder:", position.toString())
+        })
+
+        holder.itemView.setOnLongClickListener(View.OnLongClickListener()
```

Comment:
I like the use of long click here, maybe there are other applications we could use this for across the app to enhance UX